src/libraries/external/Pausable.sol

```diff
@@ -0,0 +1,108 @@
+// SPDX-License-Identifier: UNLICENSED
```

Comment:
It's not an external library so should go in the root /libraries folder imo

Reply:
put it there b/c it replicates lots of the OZ functionality but yea can move it to the root